config/docker/dependencies/centos-stream/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Build stage with Spack pre-installed and ready to be used
+FROM spack/centos-stream as builder
+
+
+# What we want to install and how we want to install it
+# is specified in a manifest file (spack.yaml)
+
+RUN which tar
+RUN tar -cf tarfilename.tar -T /dev/null
+
+RUN spack install gcc@11.3.0
```

Comment:
```suggestion
RUN spack install gcc@11.3.0+binutils
```

Reply:
I hope that this will clear up a lot of the weirdness (imo) in the current configuration. For example, on RHEL we only occasionally have needed to setup an extra LD_LIBRARY_PATH, nothing like the extent of the workarounds here. (Can send the current versions of the setup scripts)

Reply:
It didnt seem to fix the workarounds, I'll be reverting back what was previous